system/athena/athenad.py

```diff
@@ -105,8 +106,8 @@ def from_dict(cls, d: dict) -> UploadItem:
 cur_upload_items: dict[int, UploadItem | None] = {}
 
 
-def strip_bz2_extension(fn: str) -> str:
-  if fn.endswith('.bz2'):
+def strip_zst_extension(fn: str) -> str:
+  if fn.endswith('.zst'):
```

Comment:
I think sunnylink relies on the extension too. I know there's like a cleanup process for old logs and things like that and I also think that relied on the extension. If we are bz2 on sunnylink it might have an impact. So I think we should align with whats here. But I need to take a deeper look to validate before we merge this just to make sure. Adding the comment so we don't forget

Reply:
Let's add a `TODO: SP` here before we merge